spec/smoke_test/job_seekers_can_view_homepage_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'browser_test_helper'
+require 'i18n_helper'
+
+RSpec.describe 'Page availability', js: true, smoke_test: true do
+  context 'Job seeker visits vacancy page' do
+    it 'should ensure users can search and view a job vacancy page' do
+      page = Capybara::Session.new(:poltergeist)
+      page.driver.set_cookie('smoke_test', '1', domain: 'teaching-vacancies.service.gov.uk')
+
+      page.visit 'https://teaching-vacancies.service.gov.uk/'
+      expect(page).to have_content(I18n.t('jobs.heading'))
+
+      page.within '.filters-form' do
+        page.fill_in I18n.t('jobs.filters.subject'), with: 'Maths'
```

Comment:
This is going to inflate the stats we have for the number of searches performed on the website in the audit spreadsheet.  We should agree on the approach for whether to keep this search data being recorded or not.

Reply:
Resolved in fe4dbb8